mne/minimum_norm/time_frequency.py

```diff
@@ -761,7 +930,17 @@ def _compute_source_psd_epochs(
     """Generate compute_source_psd_epochs."""
     logger.info("Considering frequencies %g ... %g Hz" % (fmin, fmax))
 
-    K, sel, Vh, vertno, is_free_ori, noise_norm = _prepare_source_params(
+    if label:
+        # TO DO: add multi-label support
```

Comment:
```suggestion
        # TODO: add multi-label support
```

(convention; some syntax highlighting themes will detect "TODO" and change its color so it stands out)